src/test/Components/CreateImageWizardV2/CreateImageWizard.test.tsx

```diff
@@ -752,6 +752,57 @@ describe('Step Upload to AWS', () => {
     });
   });
 
+  describe('Step Oscap profile', () => {
+    const user = userEvent.setup();
+    const setUp = async () => {
+      ({ router } = await renderCustomRoutesWithReduxRouter(
+        'imagewizard',
+        {},
+        routes
+      ));
```

Comment:
Hmm, looking at the original tests again. Is this test new? Can't find it in the original file.

Reply:
the original tests on comliance file on Recreacte describe('Step Compliance', () => {
  const user = userEvent.setup();
  const setup = async () => {
    renderCustomRoutesWithReduxRouter('imagewizard', {}, routes);
  };
  test('create an image with None oscap profile', async () => {
    await setup();

Reply:
I see, so is this tests duplicate of this one with changed values? https://github.com/mgold1234/image-builder-frontend/blob/893c26eb5bdd7ed6c6c6c8fc950c3fb336b1726e/src/test/Components/CreateImageWizardV2/CreateImageWizard.compliance.test.tsx#L119-L170

If so I don't think we need to add here again as it's already contained in the `.compliance` test suite.